templates/shortcode-profile-editor.php

```diff
@@ -199,7 +199,7 @@
 				<input name="edd_new_user_pass1" id="edd_new_user_pass1" class="password edd-input" type="password"/>
 			</p>
 
-			<p id="edd_profile_billing_address_wrap">
+			<p id="edd_profile_pass2_wrap">
```

Comment:
I'd like to see this renamed to hold more standard with the `edd_profile_password_wrap` naming convention:
`edd_profile_password2_wrap`.